src/main/scala/codacy/plugins/DockerTest.scala

```diff
@@ -3,82 +3,68 @@ package codacy.plugins
 import java.nio.file.Path
 
 import codacy.plugins.test._
-import codacy.utils.Printer
-import org.apache.commons.io.FileUtils
-import better.files.File
+import codacy.plugins.test.multiple.MultipleTests
+import java.io.{File => JFile}
+import wvlet.log.{LogFormatter, LogLevel, LogSupport, Logger}
 
 case class Sources(mainSourcePath: Path, directoryPaths: Seq[Path])
 
-object DockerTest {
+object DockerTest extends LogSupport {
+  Logger.setDefaultFormatter(LogFormatter.SimpleLogFormatter)
+  Logger.setDefaultLogLevel(LogLevel.ALL)
 
   private lazy val config = Map("all" -> Seq(JsonTests, PluginsTests, PatternTests)) ++
     possibleTests.map { test =>
       test.opt -> Seq(test)
     }
-  private lazy val possibleTests = Seq(JsonTests, PluginsTests, PatternTests, MetricsTests)
+  private lazy val possibleTests = Seq(JsonTests, PluginsTests, PatternTests, MultipleTests, MetricsTests)
   private lazy val possibleTestNames = config.keySet
 
   def main(args: Array[String]): Unit = {
     val typeOfTests = args.headOption
     val dockerImageNameAndVersionOpt = args.drop(1).headOption
```

Comment:
side note, in the future we should use a proper library (e.g. `case-app` or something)